build.rs

```diff
@@ -70,6 +70,8 @@ fn main() {
         assert!(lsb_rel["Release"] == "14.04");
         // Setting -DTRAVIS causes the C API to be stubbed.
         c_build.define("TRAVIS", None);
+        // Set a Rust compile-time switch too.
```

Comment:
This comment isn't very useful (what does the switch do?). Could be removed?

Reply:
It makes `!cfg(travis)` true. We can kill it if you want.

Reply:
I think either remove it or explain what it does a bit more precisely (I don't mind which).

Reply:
It's standard rust stuff, so I'll kill it.